climada/hazard/tc_tracks.py

```diff
@@ -1885,6 +1890,8 @@ def _read_ibtracs_csv_single(file_name):
     tr_ds.attrs['sid'] = name
     tr_ds.attrs['orig_event_flag'] = bool(dfr['original_data']. values[0])
     tr_ds.attrs['data_provider'] = dfr['data_provider'].values[0]
+    if hasattr(dfr, 'nature'):
+        tr_ds['nature'] = ('time', dfr['nature'].values.astype('<U2'))
```

Comment:
I don't think it's necessary to support this. The legacy CSV format is a completely undocumented data type. I have never seen such a file apart from our test file. I don't know anyone who is using this functionality.

Reply:
Good point! This was just to stop the tests failing :p

Reply:
I don't understand why the tests would fail without this. Can you elaborate, please?

Reply:
Oh lol the tests were failing due to a line I added and later removed. We can definitely drop this.
I'll put together a new pull request shortly.